syftbox/server/users/auth.py

```diff
@@ -171,10 +178,40 @@ def get_user_from_email_token(
     payload = validate_email_token(server_settings, credentials.credentials)
     return payload["email"]
 
+def get_access_token(
+    credentials: Annotated[HTTPAuthorizationCredentials, Security(bearer_scheme)],
+    server_settings: Annotated[ServerSettings, Depends(get_server_settings)],
+) -> str:
+    _ = validate_access_token(server_settings, credentials.credentials)
+    return credentials.credentials
 
 def get_current_user(
     credentials: Annotated[HTTPAuthorizationCredentials, Security(bearer_scheme)],
     server_settings: Annotated[ServerSettings, Depends(get_server_settings)],
 ) -> str:
     payload = validate_access_token(server_settings, credentials.credentials)
-    return payload["email"]
\ No newline at end of file
+    return payload["email"]
+
+
+def check_line_in_file(line: str, file_path: Path) -> bool:
+    with open(file_path, 'r') as f:
+        for file_line in f:
+            if file_line == line:
+                return True
+    return False
+
+def write_line_in_file(line: str, file_path: Path):
+    with open(file_path, 'a') as f:
+        f.writelines([line])        
+
+def delete_line_from_file(line: str, file_path: Path):        
```

Comment:
We probably need to either move this to the db or lock the file somehow. The server is using multiple processes so we can have concurrent writes to the token file